ocp_resources/resource.py

```diff
@@ -701,7 +701,7 @@ def create(self, wait=False):
         self.logger.info(f"Posting {hashed_res}")
         self.logger.debug(f"\n{yaml.dump(hashed_res)}")
         resource_ = self.api.create(body=self.res, namespace=self.namespace, dry_run=self.dry_run)
-        with contextlib.suppress(ForbiddenError, AttributeError):
+        with contextlib.suppress(ForbiddenError, AttributeError, NotFoundError):
```

Comment:
In witch cases we got a `NotFoundError` here? this called after `create` and we should have the resource.

Reply:
Here: 
```
$ oc get UploadTokenRequest -A
   Error from server (NotFound): Unable to list "upload.cdi.kubevirt.io/v1beta1, Resource=uploadtokenrequests": the server could not find the requested resource (get uploadtokenrequests.upload.cdi.kubevirt.io)
```